bgpd/bgp_zebra.c

```diff
@@ -1902,7 +1902,7 @@ void bgp_zebra_route_install(struct bgp_dest *dest, struct bgp_path_info *info,
 	struct bgp_table *table = NULL;
 
 	table = bgp_dest_table(dest);
-	if (table && table->afi == AFI_L2VPN && table->safi == SAFI_EVPN)
+	if (table->afi == AFI_L2VPN && table->safi == SAFI_EVPN)
```

Comment:
Won't Coverity complain about this?

Reply:
Existing code base also does the same thing where coverity doesn't complain for accessing table ptr directly. 
Ex: https://github.com/FRRouting/frr/blob/7834c38ab38fe56a7384e77aa051316f9c6ec82b/bgpd/bgp_nexthop.c#L800

Reply:
Rajasekar - Look 2 lines up in your counter example

Reply:
so it's not a great example, but i am inclined to want to just remove all the assertions and tests for table in the code.  We really cannot get to a point where the value is NULL ( or rather something is so broken it's better to just crash ).  I would rather have a commit that addresses all of the assignments to table = bgp_dest_table and do the right thing instead of asserting or testing for it being set.

Reply:
Makes sense. 
I will work on a separate PR to do that since the current PR addresses EVPN breakage.